src/TMS/Approval/Approval.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+/* Copyright (c) 2018 Nils Haagen <nils.haagen@concepts-and-training.de> */
+
+namespace ILIAS\TMS\Approval;
+
+/**
+ *
+ */
+class Approval {
+
+	const OPEN = 1;
+	const APPROVED = 2;
+	const DECLINED = 3;
+
+	protected $id;
+	protected $booking_request_id;
+	protected $order_number;
+	protected $approval_position;
+	protected $approving_state;
+	protected $approving_usr_id;
+	protected $approving_date;
+
+	public function __construct(
+		int $id,
+		int $booking_request_id,
+		int $order_number,
+		int $approval_position,
+		int $approving_state = null,
+		int $approving_usr_id = null,
+		int $approving_date = null
+	) {
+		$this->id = $id;
+		$this->booking_request_id = $booking_request_id;
+		$this->order_number = $order_number;
+		$this->approval_position = $approval_position;
+
+		if(is_null($approving_state)) {
+			$this->approving_state = static::OPEN;
+		} else {
+			$this->checkValidState($approving_state);
+			$this->approving_state = $approving_state;
+		}
+	}
+
+	public function getId() : int {
```

Comment:
öffnende geschweifte Klammer in neue Zeile bitte. Gilt auch für alle folgenden Funktionen.